src/common/records.lua

```diff
@@ -1,7 +1,14 @@
 local utils = require(".common.utils")
+
 local records = {}
 -- defaults to landing page txid
-Records = Records or { ["@"] = { transactionId = "-k7t8xMoB8hW482609Z9F4bTFMC3MnuW8bTvTyT8pFI", ttlSeconds = 3600 } }
+Records = Records
+	or {
+		["@"] = {
+			transactionId = "-k7t8xMoB8hW482609Z9F4bTFMC3MnuW8bTvTyT8pFI",
+			ttlSeconds = 84600,
```

Comment:
```suggestion
			ttlSeconds = 900,
```
on an update, a change will take 1 day to propagate. that is too long.